apps/kampus/app/pano/__generated__/PanoFeed_viewer.graphql.ts

```diff
@@ -11,6 +11,9 @@
 import { Fragment, ReaderFragment } from 'relay-runtime';
 import { FragmentRefs } from "relay-runtime";
 export type PanoFeed_viewer$data = {
+  readonly actor: {
+    readonly displayName: string | null;
+  } | null;
```

Comment:
I just saw this in __generated__ folder, I didn't see that and thought I was going to write this here myself. After that, I've used npm run codegen, I don't know the step-by-step process.

How am I going to get the actor in the file above after adding  
`actor {displayName}` to the fragment?

Reply:
you need to run relay see this line https://github.com/kamp-us/monorepo/blob/aa6e9c6a1871c4f7d341082c4c9679f1acbb6606/apps/kampus/package.json#L10

Reply:
This is fine, no worries.